app-frontend/src/app/pages/home/home.controller.js

```diff
@@ -0,0 +1,8 @@
+class HomeController {
```

Comment:
I'm getting linting warnings from this file too, I think it needs to have a 4-space tabstop?